tests/playwright/src/utility/operations.ts

```diff
@@ -140,7 +140,7 @@ export async function deletePod(page: Page, name: string): Promise<void> {
         async () => {
           return !!(await pods.getPodRowByName(name));
         },
-        { timeout: 20000 },
+        { timeout: 20_000 },
```

Comment:
are all these number format changes needed?

Reply:
Not really, I just wanted all numbers to be have consistent format in the same file.